phinx.php

```diff
@@ -1,4 +1,10 @@
 <?php
+if(file_exists(__DIR__ . '/.env')) {
+    //Load local env file
+    $dotEnvLib = Dotenv\Dotenv::createMutable(__DIR__);
```

Comment:
@Robert-Watts are you able to get this to run? As far as I can tell, Dotenv is a composer dependency and this file does not include composer's autoload, unless Phinx is doing that on the CLI?

Reply:
Phinx loads it automatically I think (in the CLI). The problem that I was having was whenever I tried to migrate the database, it could not find the env variables. This meant that Phinx defaulted to trying to log into a database on localhost with no username and password, which is obviously incorrect. So the point of this was to force Phinx to load the ENV variables. It works on the CLI on both windows and ubuntu, however I have not tried it on the Phinx web app you can run.

Reply:
I think it does autoload composer's decencies [https://github.com/cakephp/phinx/blob/master/src/composer_autoloader.php](https://github.com/cakephp/phinx/blob/master/src/composer_autoloader.php)

Reply:
Awesome thanks for checking